plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/ResourceDetailsContentProvider.java

```diff
@@ -90,6 +90,21 @@ private void getDeploymentConfigChildren(Collection<ResourceProperty> properties
 
 	private void getBuildConfigChildren(Collection<ResourceProperty> properties, IBuildConfig config) {
 		IBuildStrategy buildStrategy = config.getBuildStrategy();
+		addStrategyTypeProperties(properties, buildStrategy);
+		properties.add(new ResourceProperty("source URL", config.getSourceURI()));
+		properties.add(new ResourceProperty("output to", config.getOutputRepositoryName()));
+		List<String> triggers = config.getBuildTriggers().stream()
+				.map(trigger -> trigger.getType().toString())
+				.collect(Collectors.toList());
+		properties.add(new ResourceProperty("build triggers", triggers));
+	}
+
+	private void addStrategyTypeProperties(Collection<ResourceProperty> properties, IBuildStrategy buildStrategy) {
+		if (buildStrategy == null) {
+			properties.add(new ResourceProperty("strategy", "<UNKNOWN>"));
```

Comment:
I noticed that in other resources from "Template Details" window, if smth is not defined i has: "(Not Provided)" text. Take a look in [ResourceDetailsLabelProvider.getStyledText](https://github.com/jbosstools/jbosstools-openshift/blob/cb4239fc371ac1e70afacea96bd54e0174dc6a79/plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/ResourceDetailsLabelProvider.java#L63). So possible it worth making it consitent and make it: `new ResourceProperty("strategy", "")` instead of `new ResourceProperty("strategy", "<UNKNOWN>")`. And then `ResourceDetailsLabelProvider` will make it "(Not Provided)"